packages/core/addon/models/table.js

```diff
@@ -135,15 +135,17 @@ function hasAllColumns(request, columns) {
         }
       }),
     dimensions = [].concat(
-      ...get(request, 'dimensions').map(dimension => {
-        let name = dimension.dimension.id,
-          defaultFields = getDefaultDimensionFields(dimension);
-        return !defaultFields.length ? name : defaultFields.map(field => canonicalizeDimension({ id: name, field }));
-      })
+      ...request.columns
```

Comment:
Maybe we should have a `dimensionColumns` cp on the request model

Reply:
added a todo for this one